src/chigame/games/models.py

```diff
@@ -6,13 +6,93 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
-    name = models.TextField()
+    # Basic information
+    name = models.CharField(max_length=255)
     description = models.TextField()
+    image = models.URLField(default="/static/images/no_picture_available.png")
+    year_published = models.PositiveIntegerField(null=True)
+
+    # Gameplay information
+    rules = models.TextField(null=True)
+
     min_players = models.PositiveIntegerField()
     max_players = models.PositiveIntegerField()
+    suggested_age = models.PositiveSmallIntegerField(null=True)
```

Comment:
Is this minimum age? I.e. if this field is 10 then is the game "10+"?

Reply:
Yes. I will add a comment.